### PR TITLE
Add method to enable PRTv3 feature flag in UI automation

### PR DESCRIPTION
**Pull request by mohitc1**
Adding method to enable PRTv3 while running UI automation

### CODE REVIEW

**Comment by shahzaibj** [comment]
Can we and should we just turn the flight on from ECS for Broker versions less than 1.0.0? This would avoid going through the UI screens to do it (which is expensive in a UI Automation test)

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 1]
```diff
@@ -321,4 +323,19 @@ public void disableMultipleWpj() {
         ThreadUtils.sleepSafely(500, TAG, "Wait before launch.");
         launch();
     }
+
+    /**
+     * Changes flight provider to local flights provider and sets
+     * PRTv3 flight flag to true.
+     */
+    public void enablePrtV3() {
```
Comment (iamgusain) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:319]: can we make it generic. 
public void setLocalFlightConfigurations(Map<string, string> flightConfigurations)

**Comment by mohitc1** [comment]
> Can we and should we just turn the flight on from ECS for Broker versions less than 1.0.0? This would avoid going through the UI screens to do it (which is expensive in a UI Automation test)

That will enable PRTv3 by default, and we would lose on testing current stack. 

But eventually, we would move to PRTv3 stack, so step of enabling the feature is transient and eventually be gone.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java (hunk 1 above)
Comment (mohitc1) [reply uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java:319]: Added generic method in addtion to this.

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java [hunk 2]
```diff
@@ -302,23 +304,35 @@ public String dismissDialog() {
...
+     */
+    public void setLocalFlight(@NonNull final String key, @NonNull final String value) {
         brokerFlightsFragment.launch();
         brokerFlightsFragment.selectLocalProvider();
-        brokerFlightsFragment.setLocalFlight(FLIGHT_FOR_WORKPLACE_JOIN_CONTROLLER, "false");
+        brokerFlightsFragment.setLocalFlight(key, value);
         ThreadUtils.sleepSafely(500, TAG, "Wait before force stop.");
         forceStop();
-        ThreadUtils.sleepSafely(500, TAG, "Wait before launch.");
+        ThreadUtils.sleepSafely(500, "TAG", "Wait before launch.");
```
Comment (fadidurah) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java]: Quotes around TAG needed?

**Review by fadidurah — APPROVED** [review]

**Review by iamgusain — APPROVED** [review]

**Pull request merged**